packages/editor/src/utils/media-upload/media-upload.js

```diff
@@ -31,7 +31,8 @@ import { __, sprintf } from '@wordpress/i18n';
  * https://bugs.chromium.org/p/chromium/issues/detail?id=227004
  *
  * @param {?Object} wpMimeTypesObject Mime type object received from the server.
- *                                    Extensions are keys separated by '|' and values are mime types associated with an extension.
+ *                         	// Create upload payload
```

Comment:
Hi @antpb, I'm not seeing the relation between this changes and the PR, were they intentional or I'm missing something?